templates/frontend/objects/chapter.tpl

```diff
@@ -0,0 +1,501 @@
+				datePublished=$publication->getData('datePublished')|date_format:$dateFormatShort
+				urlRecentVersion=$latestVersionUrl|escape
+			}
+		</div>
+	{/if}
+
+	<h1 class="title">
+		{$chapter->getLocalizedFullTitle()|escape}
+	</h1>
+
+	<div class="row">
+		<div class="main_entry">
+
+			{* Author list *}
+			{include file="frontend/components/authors.tpl" authors=$chapterAuthors}
+
+			{* DOI (requires plugin) *}
+			{foreach from=$pubIdPlugins item=pubIdPlugin}
+				{if $pubIdPlugin->getPubIdType() != 'doi'}
+					{continue}
+				{/if}
+				{assign var=pubId value=$chapter->getStoredPubId($pubIdPlugin->getPubIdType())}
+
+				{if $pubId}
+					{assign var="doiUrl" value=$pubIdPlugin->getResolvingURL($currentPress->getId(), $pubId)|escape}
+					<div class="item doi">
+						<span class="label">
+							{translate key="plugins.pubIds.doi.readerDisplayName"}
+						</span>
+						<span class="value">
+							<a href="{$doiUrl}">
+								{$doiUrl}
+							</a>
+						</span>
+					</div>
+				{/if}
+			{/foreach}
+
+			{* Keywords *}
+			{if !empty($publication->getLocalizedData('keywords'))}
+			<div class="item keywords">
+				<h2 class="label">
+					{capture assign=translatedKeywords}{translate key="common.keywords"}{/capture}
+					{translate key="semicolon" label=$translatedKeywords}
+				</h2>
+				<span class="value">
+					{foreach name="keywords" from=$publication->getLocalizedData('keywords') item=keyword}
+						{$keyword|escape}{if !$smarty.foreach.keywords.last}, {/if}
+					{/foreach}
+				</span>
+			</div>
+			{/if}
+
+			{* Abstract *}
+			<div class="item abstract">
+				<h2 class="label">
+					{translate key="submission.synopsis"}
+				</h2>
+				<div class="value">
+					{$chapter->getLocalizedData('abstract')|strip_unsafe_html}
+				</div>
+			</div>
+
+			{call_hook name="Templates::Catalog::Chapter::Main"}
+
+			{* Determine if any authors have biographies to display *}
+			{assign var="hasBiographies" value=0}
+			{foreach from=$chapterAuthors item=author}
+				{if $author->getLocalizedBiography()}
+					{assign var="hasBiographies" value=$hasBiographies+1}
+				{/if}
+			{/foreach}
+			{if $hasBiographies}
+				<div class="item author_bios">
+					<h2 class="label">
+						{if $hasBiographies > 1}
+							{translate key="submission.authorBiographies"}
+						{else}
+							{translate key="submission.authorBiography"}
+						{/if}
+					</h2>
+					{foreach from=$chapterAuthors item=author}
+						{if $author->getLocalizedBiography()}
+							<div class="sub_item">
+								<div class="label">
+									{if $author->getLocalizedAffiliation()}
+										{capture assign="authorName"}{$author->getFullName()|escape}{/capture}
+										{capture assign="authorAffiliation"}<span class="affiliation">{$author->getLocalizedAffiliation()|escape}</span>{/capture}
+										{translate key="submission.authorWithAffiliation" name=$authorName|escape affiliation=$authorAffiliation|escape}
```

Comment:
The `$authorAffiliation` here is double-escaped, which leads to the `<span class="affiliation">` tag appearing in the author bios.

![author-bios](https://user-images.githubusercontent.com/2306629/140784798-245fd25b-2b09-4744-81bb-5ecb1d7b1999.png)

Reply:
Since this changes [https://github.com/pkp/omp/commit/d1ced3804f910f31cc40acb76ab00054bde53e53](https://github.com/pkp/omp/commit/d1ced3804f910f31cc40acb76ab00054bde53e53) it's the same in monograph_full.tpl. Should I change this too or do you want discuss this first with @asmecher?

Reply:
Good spot! Yes, can you fix this in both places? If I understand correctly, the `|escape` filters can be removed from this line, which appears in two plces in `monograph_full.tpl`:

```
{translate key="submission.authorWithAffiliation" name=$authorName|escape affiliation=$authorAffiliation|escape}
```